### PR TITLE
Migrate vanilla JS example to `todomvc-app-css`

### PR DESCRIPTION
**Pull request by arthurvr**
Also switches it over to NPM.

### CODE REVIEW

File: examples/vanillajs/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "todomvc-vanillajs",
```
Comment (sindresorhus) [inline examples/vanillajs/package.json]: this is not needed

File: examples/vanillajs/package.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  "private": "true",
```
Comment (sindresorhus) [inline examples/vanillajs/package.json]: should be a boolean

File: examples/vanillajs/package.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  "description": "TodoMVC implementation in Vanilla JavaScript",
```
Comment (sindresorhus) [inline examples/vanillajs/package.json]: not needed

File: examples/vanillajs/package.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  "dependencies": {
+    "todomvc-common": "~1.0.1",
```
Comment (sindresorhus) [inline examples/vanillajs/package.json]: `~` => `^`

File: examples/vanillajs/test/SpecRunner.html [hunk 5]
```diff
@@ -3,9 +3,9 @@
 <head>
   <title>Jasmine Spec Runner</title>
 
-  <link rel="stylesheet" type="text/css" href="../bower_components/jasmine/lib/jasmine-core/jasmine.css">
-  <script type="text/javascript" src="../bower_components/jasmine/lib/jasmine-core/jasmine.js"></script>
-  <script type="text/javascript" src="../bower_components/jasmine/lib/jasmine-core/jasmine-html.js"></script>
+  <link rel="stylesheet" type="text/css" href="../node_modules/jasmine/lib/jasmine-core/jasmine.css">
+  <script type="text/javascript" src="../node_modules/jasmine/lib/jasmine-core/jasmine.js"></script>
+  <script type="text/javascript" src="../node_modules/jasmine/lib/jasmine-core/jasmine-html.js"></script>
```
Comment (sindresorhus) [inline examples/vanillajs/test/SpecRunner.html]: not yours, but since you're chaning. the `type` attribute on these can be removed.

Comment (arthurvr) [inline examples/vanillajs/test/SpecRunner.html]: updated all of that.

**Comment by arthurvr** [comment]
@sindresorhus Updated PR with all of that.

File: examples/vanillajs/package.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{
+  "private": true,
+  "dependencies": {
+    "todomvc-common": "^1.0.1",
+    "todomvc-app-css": "^1.0.1"
+  },
+  "devDependencies": {
+    "jasmine-core": "~0.3.0"
```
Comment (sindresorhus) [inline examples/vanillajs/package.json]: this too

File: examples/vanillajs/test/SpecRunner.html [hunk 7]
```diff
@@ -3,9 +3,9 @@
...
 
-  <link rel="stylesheet" type="text/css" href="../bower_components/jasmine/lib/jasmine-core/jasmine.css">
-  <script type="text/javascript" src="../bower_components/jasmine/lib/jasmine-core/jasmine.js"></script>
-  <script type="text/javascript" src="../bower_components/jasmine/lib/jasmine-core/jasmine-html.js"></script>
+  <link rel="stylesheet" href="../node_modules/jasmine/lib/jasmine-core/jasmine.css">
+  <script src="../node_modules/jasmine/lib/jasmine-core/jasmine.js"></script>
+  <script src="../node_modules/jasmine/lib/jasmine-core/jasmine-html.js"></script>
 
   <!-- include spec files here... -->
   <script type="text/javascript" src="ControllerSpec.js"></script>
```
Comment (sindresorhus) [inline examples/vanillajs/test/SpecRunner.html]: and this

Comment (arthurvr) [inline examples/vanillajs/test/SpecRunner.html]: updated

**Comment by sindresorhus** [comment]
od3vvoq

**Pull request merged**